arch/arm/src/cxd56xx/cxd56_spi.c

```diff
@@ -871,7 +886,7 @@ static void spi_exchange(struct spi_dev_s *dev, const void *txbuffer,
   size_t dmath = 0;
 #endif
 
-  if (priv->dmaenable && dmath < nwords)
+  if (priv->txdmach && priv->rxdmach && dmath < nwords)
     {
```

Comment:
Yes, because txdmach and rxdmach doesn't initialize at runtime anymore, so dmaenable which is used to identify the initialization result doesn't need too.